workos/user_management.py

```diff
@@ -476,6 +478,9 @@ def get_authorization_url(
             params["login_hint"] = login_hint
         if state is not None:
             params["state"] = state
+        if code_challenge:
+            params["code_challenge"] = code_challenge
+            params["code_challenge_method"] = "S256"
```

Comment:
quick follow up - since this is a static value, does the client need to pass it in at all?

Reply:
It's listed in the API reference as a param for curl, so I included it here